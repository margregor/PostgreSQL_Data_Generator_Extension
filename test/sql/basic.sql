CREATE EXTENSION database_data_generator;
SELECT * FROM generate_data(10) AS t("date" DATE, "integer" BIGINT, "float" REAL, "Miasto Zamieszkania" TEXT);
DROP EXTENSION database_data_generator;