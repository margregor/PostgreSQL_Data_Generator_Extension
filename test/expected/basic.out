CREATE EXTENSION database_data_generator;
SELECT * FROM generate_data(10) AS t("date" TEXT, "integer" TEXT, "address" TEXT);
    date    | integer |        address        
------------+---------+-----------------------
 2025-04-13 | 123     | 123 Fake St, Faketown
 2025-04-13 | 123     | 123 Fake St, Faketown
 2025-04-13 | 123     | 123 Fake St, Faketown
 2025-04-13 | 123     | 123 Fake St, Faketown
 2025-04-13 | 123     | 123 Fake St, Faketown
 2025-04-13 | 123     | 123 Fake St, Faketown
 2025-04-13 | 123     | 123 Fake St, Faketown
 2025-04-13 | 123     | 123 Fake St, Faketown
 2025-04-13 | 123     | 123 Fake St, Faketown
 2025-04-13 | 123     | 123 Fake St, Faketown
(10 rows)

DROP EXTENSION database_data_generator;
